src/components/ExternalLinkIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default function ExternalLinkIcon() {
+  return (
+    <svg viewBox='0 0 24 24' width='13.5' height='13.5' aria-hidden='true' style={{ marginLeft: '0.3rem' }}>
+      <path fill='currentColor' d='M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z'
+      ></path>
+      <path fill='currentColor' d='M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z'
+      ></path>
+      <path fill='currentColor' d='M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z'
+      ></path>
+    </svg>
```

Comment:
Was this icon custom made by you?

Reply:
No, this was taken directly from the navbar `Forum` external link icon, I think auto-generated by docusaurus.